### PR TITLE
feat: Move DEFAULT_STARTING_ANCHOR_ROOTS to Constants library

### PR DESCRIPTION
**Pull request by maurelian**

### CODE REVIEW

**Comment by maurelian** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12320?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#12291** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12291?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12320** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12320?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#12287** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12287?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12260** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12260?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#12261](https://github.com/ethereum-optimism/optimism/pull/12261) <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12261?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#12283** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/12283?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @maurelian and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/contracts-bedrock/src/libraries/Constants.sol [hunk 1]
```diff
@@ -50,4 +52,14 @@ library Constants {
         });
         return config;
     }
+
+    function DEFAULT_STARTING_ANCHOR_ROOTS() internal pure returns (IAnchorStateRegistry.StartingAnchorRoot[] memory) {
```
Comment (mds1) [inline packages/contracts-bedrock/src/libraries/Constants.sol]: Hmm one thing that I don't love is that this lives in `src`, but is not used by any contracts in `src/`

Comment (maurelian) [reply packages/contracts-bedrock/src/libraries/Constants.sol]: [df735c7](https://github.com/ethereum-optimism/optimism/pull/12320/commits/df735c7b79408f7b8b37aac903bdbec332487ef0)

File: packages/contracts-bedrock/src/libraries/Constants.sol [hunk 2]
```diff
@@ -2,6 +2,8 @@
 pragma solidity ^0.8.0;
 
 import { IResourceMetering } from "src/L1/interfaces/IResourceMetering.sol";
+import { IAnchorStateRegistry } from "src/dispute/interfaces/IAnchorStateRegistry.sol";
+import { GameTypes, OutputRoot, Hash } from "src/dispute/lib/Types.sol";
```
Comment (mds1) [inline packages/contracts-bedrock/src/libraries/Constants.sol:6]: delete?

**Review by mds1 — APPROVED** [review]

File: packages/contracts-bedrock/src/libraries/Constants.sol (hunk 2 above)
Comment (maurelian) [reply packages/contracts-bedrock/src/libraries/Constants.sol:6]:
```suggestion
```

**Pull request closed without merging**